server/plugin/http.go

```diff
@@ -21,7 +24,7 @@ type Error struct {
 	Message string `json:"message"`
 }
 
-var ErrorConnectionRefused = fmt.Errorf("unable to make a connection to the specified ServiceNow instance")
+var ErrorContentTypeNotJSON = fmt.Errorf("content type of the response is not JSON")
```

Comment:
sounds better i think
```suggestion
var ErrorContentTypeNotJSON = fmt.Errorf("response content type is not JSON")
```